source/2020-04-24-the-ember-times-issue-145.md

```diff
@@ -26,14 +27,20 @@ I think articles like this will help you build your developer skillset. There is
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Ember Styleguide 4.0.0 Release 🔥](https://github.com/ember-learn/ember-styleguide/releases/tag/v4.0.0)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+A new Ember Styleguide release was cut today 🛳 and it represents a massive amount of work by a number of contributors over the last year. 372 commits, 719 files and 13 contributors to be exact! 😱
```

Comment:
```suggestion
A new Ember Styleguide release was cut today 🛳 and it represents a massive amount of work by a number of contributors over the last year: 372 commits, 719 files and 13 contributors to be exact! 😱 Check out the app [here on the web](https://ember-styleguide.netlify.app/) as well.
```

Not sure of the wording, but thought it may make sense to add the web link, thoughts?